packages/web/app/src/pages/target-settings.tsx

```diff
@@ -540,6 +559,8 @@ const ConditionalBreakingChanges = (props: {
           projectSlug: props.projectSlug,
           targetSlug: props.targetSlug,
           ...values,
+          requestCount: values.requestCount ?? 1,
+          percentage: values.percentage ?? 0,
```

Comment:
I opted to default these here instead of in `Yup` so that users dont see these values appear when clicking submit.

Alternatively, I think it's an okay experience to set the default value onBlur like I did for the [requestCount](https://github.com/graphql-hive/console/pull/6254/files#diff-40476c14d0063812ffd974b7048fe7d14f916776fcca4a67fc371821bc63c7f5R675)